.gitignore

```diff
@@ -13,7 +13,7 @@ __pycache__/*
 tcf_website/migrations/*
 !tcf_website/migrations/__init__.py
 # Environment variables
-.env*
+.env
```

Comment:
Is this an intended change? Because it shouldn't be - the `*` is there because there are occasionally multiple `.env` files that can be used.